Assets/Scripts/Camera/Camera.cs

```diff
@@ -0,0 +1,67 @@
+using UnityEngine;
+
+namespace Cosmobot
+{
+    public class Camera : MonoBehaviour
+    {
+        public float sensX;
+        public float sensY;
+        public float cameraChangeY;
+        public float cameraChangeZ;
+        public KeyCode cameraChangeKey;
+        private float _xRotation;
+        private float _yRotation;
+        private bool _isFirstPerson = true;
+        public Transform playerObject;
+        public Transform cameraHolder;
+        
+        private void Start()
+        {
+            Cursor.lockState = CursorLockMode.Locked;
+            Cursor.visible = false;
+            transform.position = cameraHolder.position;
+        }
+
+        private void Update()
+        {
+            var mouseX = Input.GetAxisRaw("Mouse X") * Time.deltaTime * sensX;
+            var mouseY = Input.GetAxisRaw("Mouse Y") * Time.deltaTime * sensY;
```

Comment:
Use new InputSystem. It is already configured in project, you only need to add camera movement actions in `Assets/ProjectConfig/DefaultInputActions` asset.

I like approach with generated interface: [Using Input Action Assets](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/ActionAssets.html#using-input-action-assets)
```cs
// implement generated interface
public class Camera : MonoBehaviour, DefaultInputActions.IPlayerMovementActions
    {
        DefaultInputActions actions;
        
        // override method from interface
        // replace "OnMovement" with your Action name
        public void OnMovement(InputAction.CallbackContext context) {
            Debug.Log(context.ReadValue<Vector2>()); // read value
        }

        void OnEnable() {
            if(actions == null) {
                actions = new DefaultInputActions();
                actions.PlayerMovement.SetCallbacks(this);
            }
            actions.PlayerMovement.Enable();
        }
        void OnDisable() {
            actions.PlayerMovement.Disable();
        }
        //...
}
```
  
But, there is also more popular approach with `PlayerInput` Asset: [Using Action Assets with PlayerInput](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/ActionAssets.html#using-action-assets-with-playerinput).

InputSystem tutorial: https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/ActionsEditor.html